product/src/main/java/com/product/api/controller/CtrlProduct.java

```diff
@@ -0,0 +1,36 @@
+package com.product.api.controller;
+import com.product.Almacen;
+import com.product.api.service.SvcCategory;
+import com.product.api.service.SvcCategoryImp;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+
+
+
+@RestController
+@RequestMapping("/")
```

Comment:
Utilizaremos directamente la ruta para la clase, que debes renombrar a `CtrlCategory.java`
```suggestion
@RequestMapping("/category")
```